lib/upfluence/endpoint/api_endpoint.rb

```diff
@@ -38,6 +39,18 @@ def respond_with(resource, *args)
           halt [status, result] || 404
         end
       end
+
+      def json_params
+        begin
+          JSON.parse(request.body.read)
```

Comment:
Because you used `active_model_serializer` in #2, it's already un underlied dependency